### PR TITLE
DOCS-1734: Document batch data upload through app

### PR DESCRIPTION
**Pull request by sguequierre**
* Add data client api and mobile app upload to Data Export page -- data client api because it's not mentioned at all and hard to find from data section. Can revert if this is purposeful?
* Add mention of data upload to app section in fleet

### CODE REVIEW

File: docs/fleet/_index.md [hunk 1]
```diff
@@ -77,6 +77,6 @@ Machine [configuration](machines/#configuration) and machine [code](/build/progr
 <br>
 
 In addition to the [Viam app](https://app.viam.com), the fully featured web application where you can access all fleet management tools, there is a Viam mobile app.
-The mobile app is a convenient way to see if your machine is online, access the [control interface](/fleet/machines/#control), and check machine [logs](/fleet/machines/#logs).
+The mobile app is a convenient way to see if your machine is online, access the [control interface](/fleet/machines/#control), check machine [logs](/fleet/machines/#logs), and [upload images to the cloud]().
```
Comment (sguequierre) [inline docs/fleet/_index.md:80]:
```suggestion
The mobile app is a convenient way to see if your machine is online, access the [control interface](/fleet/machines/#control), check machine [logs](/fleet/machines/#logs), and [upload images to the cloud](/data/export/#upload-images-with-the-viam-mobile-app).
```

File: docs/data/export.md [hunk 2]
```diff
@@ -37,6 +42,28 @@ Then, to export data from the data management service in the cloud:
...
+
+Click on the method name for more information.
+
+## Upload images with the Viam mobile app
+
+Upload images as machine data straight from your phone, skipping the normal data capture and cloud synchronization process, through the [Viam mobile app](/fleet/#the-viam-mobile-app).
+This is useful if you want to capture images for training machine learning models on the go.
+
+1. Select **Home** on the mobile app, choose your organization, and select a location and machine.
+2. Then, click the menu button **...** in the upper right corner.
```
Comment (sguequierre) [inline docs/data/export.md:61]:
```suggestion
2. Click the menu button marked "**...**" in the upper right corner.
```

File: docs/data/export.md [hunk 3]
```diff
@@ -37,6 +42,28 @@ Then, to export data from the data management service in the cloud:
...
+3. Click **Upload Images**.
+4. Then, select each image you want to upload, and click **Add**.
```
Comment (sguequierre) [inline docs/data/export.md:63]:
```suggestion
4. Select each image you want to upload, and click **Add**.
```

File: docs/data/export.md [hunk 4]
```diff
@@ -37,6 +42,28 @@ Then, to export data from the data management service in the cloud:
...
+
+Upload images as machine data straight from your phone, skipping the normal data capture and cloud synchronization process, through the [Viam mobile app](/fleet/#the-viam-mobile-app).
+This is useful if you want to capture images for training machine learning models on the go.
+
+1. Select **Home** on the mobile app, choose your organization, and select a location and machine.
+2. Click the menu button marked "**...**" in the upper right corner.
+3. Click **Upload Images**.
+4. Select each image you want to upload, and click **Add**.
+
+As for metadata, these images are associated with this machine part you selected, and aren't associated with a component or method.
```
Comment (npentrel) [inline docs/data/export.md:65]:
```suggestion
As for metadata, these images are associated with the machine part you selected, and aren't associated with a component or method.
```

File: docs/data/export.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Export Data"
-linkTitle: "Export Data"
-description: "Export data from the Viam app to your local computer using the Viam CLI."
+title: "Upload and Export Data"
+linkTitle: "Upload and Export Data"
+description: "Download data from and upload data to the Viam app from your local computer using the data client API, Viam CLI, or Viam mobile app."
```
Comment (npentrel) [inline docs/data/export.md:4]: I think we should create upload as its own page - up to you and @JessamyT whether you want to merge this PR and create the new page later or create the page now.

Comment (sguequierre) [reply docs/data/export.md:4]: I would be good with creating the new page later 😄

Comment (npentrel) [reply docs/data/export.md:4]: I think that does make sense because by itself this may look odd

Comment (npentrel) [reply docs/data/export.md:4]: can you comment this pr or the page link on the jira ticket for the upload page?

**Review by npentrel — APPROVED** [review]

File: docs/data/export.md (hunk 5 above)
Comment (sguequierre) [reply docs/data/export.md:4]: @npentrel Sorry, I can't find that ticket! Which one are you referring to?

**Comment by npentrel** [comment]
@sguequierre is this ready to merge?

**Pull request merged**